frontend/src/components/pages/ManageStoryTranslationPage.tsx

```diff
@@ -156,15 +194,27 @@ const ManageStoryTranslationPage = () => {
             <Heading size="sm" marginTop="24px" marginBottom="18px">
               Title
             </Heading>
-            <Input type="title" value={title || ""} />
+            <Input
+              type="title"
+              value={tempTitle || ""}
+              onChange={(event) => setTempTitle(event.target.value)}
+            />
             <Heading size="sm" marginTop="24px" marginBottom="18px">
               Description
             </Heading>
-            <Textarea type="description" value={description || ""} />
+            <Textarea
+              type="description"
+              value={tempDescription || ""}
+              onChange={(event) => setTempDescription(event.target.value)}
+            />
             <Heading size="sm" marginTop="24px" marginBottom="18px">
               YouTube Link
             </Heading>
-            <Input type="youtubeLink" value={youtubeLink} />
+            <Input
+              type="youtubeLink"
+              value={tempYoutubeLink}
+              onChange={(event) => setTempYoutubeLink(event.target.value)}
+            />
           </FormControl>
           <Heading size="sm" marginTop="24px">
             Delete Story Translation
```

Comment:
mind removing the TODOs on lines 226 and 267 for me? My bad, i should've removed them in my pr